lib/system/mmdisp.nim

```diff
@@ -45,7 +45,7 @@ else:
 
 proc raiseOutOfMem() {.noinline.} =
   if outOfMemHook != nil: outOfMemHook()
-  cstderr.rawWrite("out of memory\n")
+  writeToStdErr("out of memory")
```

Comment:
```suggestion
  writeToStdErr("out of memory\n")
```

Reply:
Must have just been for style, the next line is `quit` so the buffer should flush regardless.

Reply:
adding a newline isn't the same as flushing; this changes semantics and resulting stderr output